ansible/install-apps.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_repo_list_file: /etc/apt/sources.list.d/google-chrome.list
+  tasks:
+  - stat:
+      path: "{{ apt_repo_list_file }}"
+    register: google_apt_exists
+    ignore_errors: yes
+  - name: dir
+    copy:
```

Comment:
It's nice that you've figured out how to use copy but what you actually need is to use `apt_repository` module.